packages/oauth/jwk/src/keyset.ts

```diff
@@ -162,6 +162,7 @@ export class Keyset<K extends Key = Key> implements Iterable<K> {
 
       // Skip negotiation if a specific "alg" was provided
       if (typeof alg === 'string') return [key, alg]
+      else if (alg?.length === 1) return [key, alg[0]!]
```

Comment:
Is this perhaps from a different line of work?

Reply:
yup. reverted.